code/_globalvars/phobias.dm

```diff
@@ -65,6 +65,7 @@ GLOBAL_LIST_INIT(phobia_mobs, list(
 		/mob/living/basic/chicken,
 		/mob/living/basic/parrot,
 		/mob/living/basic/pet/penguin,
+		/mob/living/simple_animal/hostile/retaliate/goose
```

Comment:
> > This is also something you'll want to open a PR [upstream](https://github.com/tgstation/tgstation) because this is a TG file.
> 
> How do i uh... do that

Same as here, you fork the TG git, make the edits, and then create a PR